plugins/plugin-chart-echarts/src/Graph/controlPanel.tsx

```diff
@@ -0,0 +1,107 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import { t } from '@superset-ui/core';
+import { formatSelectOptions, sections } from '@superset-ui/chart-controls';
+import { DEFAULT_FORM_DATA } from './types';
+
+const { layout, roam, draggable, selectedMode, showSymbolThreshold } = DEFAULT_FORM_DATA;
+
+export default {
+  controlPanelSections: [
+    sections.legacyRegularTime,
+    {
+      label: t('Query'),
+      expanded: true,
+      controlSetRows: [['groupby'], ['metric'], ['adhoc_filters'], ['row_limit']],
+    },
+    {
+      label: t('Chart Options'),
+      expanded: true,
+      controlSetRows: [
+        [
+          {
+            name: 'layout',
+            config: {
+              type: 'SelectControl',
+              renderTrigger: true,
+              label: t('Graph Layout'),
+              default: layout,
```

Comment:
nit: I think the code would read clearer if you just do:

```suggestion
              default: DEFAULT_FORM_DATA.layout,
```

Because then you can know for sure where these default values come from without scrolling up and down.